plugins/pulp_rpm/plugins/db/models.py

```diff
@@ -121,8 +121,8 @@ class NonMetadataPackage(UnitMixin, FileContentUnit):
     :ivar checksum: The checksum of the package.
     :type checksum: mongoengine.StringField
 
-    :ivar signature: The signature of the package.
-    :type signature: mongoengine.StringField
+    :ivar signing_key: The signing key of the package.
```

Comment:
Here it would also be good to specify that it's just the 8-character abbreviated fingerprint.